etna/packages/etna-js/upload/reducers/upload-reducer.jsx

```diff
@@ -1,87 +1,12 @@
-import { fileKey } from '../../utils/file';
```

Comment:
The upload reducer gets major simplification; it basically just reflects the state passed to it from update_upload.  The worker manages its own state directly.